.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
 
 ### Steps
 
-<!-- please tick steps this PR performs (if something is not necessary, please tick anyway to indicate you considered it) -->
+<!--please tick steps this PR performs (if something is not necessary, please remove it) -->
 
 - [ ] added implementation
 - [ ] added / updated tests
 - [ ] added / updated documentation
 - [ ] extended changelog
```

Comment:
We don't really have a "PRs extend the changelog" policy here, maybe remove this step?

Reply:
fixed